project/BalboaAgent.scala

```diff
@@ -1,31 +1,19 @@
 import Dependencies._
-import com.typesafe.sbt.SbtNativePackager._
 import com.typesafe.sbt.packager.debian
 import com.typesafe.sbt.packager.debian.DebianPlugin.autoImport._
 import sbt.Keys._
 import sbt._
-import sbtassembly.Plugin.AssemblyKeys._
 
 object BalboaAgent {
-  lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings(assembly = true) ++ Seq(
-    mainClass in assembly := Some("com.socrata.balboa.agent.BalboaAgent"),
+  lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings ++ Seq(
+    mainClass in sbtassembly.AssemblyKeys.assembly := Some("com.socrata.balboa.agent.BalboaAgent"),
     libraryDependencies <++= scalaVersion { libraries(_) },
-    jarName in assembly := s"${name.value}-assembly.jar",
```

Comment:
Yeah Ill just change the debian to a plain old assembly jar to coincide with the way we are doing things.